### PR TITLE
dev: use events api to open model in React app - EXAMPLE

### PR DESCRIPTION
**Pull request by luqven**
> EXAMPLE PR DO NOT MERGE

This PR cherry-picks #60 and #62 and removes some unused files. It also adjusts the build scripts to account for new file names.

Most of the changes from #62 are integrated into `index-sidebar.tsx` and `index-breakout.tsx`.

### CODE REVIEW

File: frontend/src/index-breakout.tsx [hunk 1]
```diff
@@ -4,38 +4,35 @@ import { App } from "./components";
...
+    ReactDOM.render(
+      <React.StrictMode>
+        <App />
+      </React.StrictMode>,
+      rootEditorElement
+    );
+    const openButtonEl = document.querySelector(".ix-selection");
+    if (openButtonEl) {
+      openButtonEl.addEventListener("click", handleSelect);
+    }
```
Comment (frederickfogerty) [inline frontend/src/index-breakout.tsx:26]: I know this is an example so maybe you were going to do this anyway, but I think we should use props for this event handler. i.e: 

```suggestion
    ReactDOM.render(
      <React.StrictMode>
        <App onValueSet={handleSelect} />
      </React.StrictMode>,
      rootEditorElement
    );
```

File: frontend/src/index-sidebar.tsx [hunk 2]
```diff
@@ -3,39 +3,87 @@ import ReactDOM from "react-dom";
...
-      <App />
-    </React.StrictMode>,
-    rootEditorElement
+export const createSidebarApp = () => {
+  let localization: any;
+  let buttonEl: any;
+
+  subscribe(
+    "sfcc:ready",
+    async ({
```
Comment (frederickfogerty) [inline frontend/src/index-sidebar.tsx:17]: Just wondering why this is `async`?

File: frontend/src/index-sidebar.tsx [hunk 3]
```diff
@@ -3,39 +3,87 @@ import ReactDOM from "react-dom";
...
+      ReactDOM.render(
+        <React.StrictMode>
+          <App />
+        </React.StrictMode>,
+        rootEditorElement
+      );
+
+      // Obtain DOM elements and apply event handlers
+      buttonEl = document.querySelector(".ix-btn");
+      buttonEl.addEventListener("click", handleBreakoutOpen);
```
Comment (frederickfogerty) [inline frontend/src/index-sidebar.tsx:51]: Same as above

```suggestion
      ReactDOM.render(
        <React.StrictMode>
          <App onBreakoutOpen={handleBreakoutOpen} />
        </React.StrictMode>,
        rootEditorElement
      );
```

**Review by frederickfogerty — COMMENTED** [review]
I know this is a draft/example, but nice work 👏 and also looks almost ready to merge

**Comment by luqven** [comment]
replaced by https://github.com/imgix/sf-commerce-cloud/pull/66

**Pull request closed without merging**